lte/gateway/c/oai/tasks/gtpv1-u/gtpv1u.h

```diff
@@ -180,4 +180,9 @@ int gtpv1u_add_s8_tunnel(
     struct in_addr pgw, uint32_t i_tei, uint32_t o_tei, uint32_t pgw_i_tei,
     uint32_t pgw_o_tei, Imsi_t imsi, struct ip_flow_dl* flow_dl,
     uint32_t flow_precedence_dl);
+
+int gtpv1u_del_s8_tunnel(
+    struct in_addr enb, struct in_addr pgw, struct in_addr ue,
+    struct in6_addr* ue_ipv6, uint32_t i_tei, uint32_t o_tei,
+    struct ip_flow_dl* flow_dl);
```

Comment:
why do you need to define this?

Reply:
Defined it to resolve the warning.

Reply:
While fixing the warning, I defined it

Reply:
you can remove the definition from gtpv1u.c if it was complaining about that. Otherwise, could you post what the warning is?

Reply:
oh, you are using it in s8 task, too?

Reply:
According to PR, https://github.com/magma/magma/pull/5420 
openflow_add_s8_tunnel() and openflow_del_s8_tunnel(), apis are exposed for adding and deleting s8 tunnels. Modified the code to use these apis instead of "gtp_tunnel_ops->del_s8_tunnel" and "gtp_tunnel_ops->add_s8_tunnel"